docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/on-premises.md

```diff
@@ -58,18 +58,18 @@ showAsideToc: true
 
 </ul>
 
-
 ## Platform Node Preparation
 
-*   Hardware requirements
-    *   Nodes: 1 Virtual Machine 
-    *   Cores: 8 Cores
-    *   RAM: 16 GB
-*   Storage Disk:  100GB (minimum) (confirm SSD)
-*   Docker Engine: supported version 19.03.n. If not installed, see Installing Docker in [airgapped](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/).
-*   The following ports should be open on the Yugabyte Platform host:
-    *   8800 – HTTP access to the Replicated UI
-    *   80 – HTTP access to the Yugabyte Platform console
-    *   443 - HTTPs access to the Yugabyte Platform console
-    *   22 – SSH
-*   Ability to connect from the Yugabyte Platform host to all YugabyteDB data nodes. 
+* Hardware requirements
+  * Nodes: 1 Virtual Machine 
+  * Cores: 8 Cores
+  * RAM: 16 GB
+* Storage Disk:  100GB (minimum) (confirm SSD)
+* Docker Engine: supported version 19.03.n. If not installed, see Installing Docker in [airgapped](https://www.replicated.com/docs/kb/supporting-your-customers/installing-docker-in-airgapped/).
+* The following ports should be open on the Yugabyte Platform host:
+  * 8800 – HTTP access to the Replicated UI
+  * 80 – HTTP access to the Yugabyte Platform console
+  * 443 - HTTPs access to the Yugabyte Platform console
+  * 22 – SSH
+* If you will be manually provisioning your nodes, the following additional TCP ports must be accessible: 7000, 7100, 9000, 9100, 11000, 12000, 9300, 9042, and 6379. (For more information on ports used by YugabyteDB, refer to [Default ports](../../../../reference/configuration/default-ports).)
```

Comment:
This last sentence belongs in the "prepare nodes (on-prem)" - not the platform prepare the environment section.

Reply:
https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/prepare-on-prem-nodes/

Reply:
When you move these ports to the on-prem prepare nodes section, I recommend the following language.

---

"The following ports must be opened for intra-cluster communication - they need not be exposed to your application, only to other nodes in the cluster and the platform node:

7100 - master RPC
9100 - tserver RPC
11000 - yedis API
12000 - ycql API
13000 - ysql API

The following ports must be exposed for intra-cluster communication, and it is recommended that you additionally expose these ports to administrators or monitoring users of the system, as these ports provide valuable diagnostic troubleshooting and metrics:

9300 - Prometheus metrics
7000 - master HTTP endpoint
9000 - tserver HTTP endpoint


The following nodes must be available to your application or any user attempting to connect to the yugabyte DB, in addition to intra-node communication. 

5433 - ysql server
9042 - ycql server
6379 - yedis server